broker-node/app/Http/Controllers/UploadSessionController.php

```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers;
 
+use App\DataMapping;
```

Comment:
is this persistent?

Reply:
It follows the ActiveRecord pattern. It has methods attached that can persist the models.